### PR TITLE
WT-2522: Incorrect format code in message

### PR DESCRIPTION
**Pull request by keithbostic**
@agorrod, for your review/consideration.

### CODE REVIEW

File: src/os_win/os_fs.c [hunk 1]
```diff
@@ -361,8 +361,8 @@ __win_handle_read(
 		if (!ReadFile(fh->filehandle, addr, chunk, &nr, &overlapped))
 			WT_RET_MSG(session,
 			    nr == 0 ? WT_ERROR : __wt_getlasterror(),
-			    "%s: handle-read: ReadFile: failed to read %"
-			    WT_SIZET_FMT " bytes at offset %" PRIuMAX,
+			    "%s: handle-read: ReadFile: failed to read %lu "
```
Comment (agorrod) [inline src/os_win/os_fs.c]: @keithbostic I think ``%lu`` expects a 64 bit value, whereas a ``DWORD`` is 32 bits - am I missing something?

Comment (keithbostic) [inline src/os_win/os_fs.c]: @agorrod, I believe a `DWORD` is typedef'd as an `unsigned long` on Windows, and both are 4B.

https://msdn.microsoft.com/en-us/library/aa383751(VS.85).aspx

Comment (agorrod) [inline src/os_win/os_fs.c]: D'oh. Thanks @keithbostic I misread the description in the ticket - I'd thought a capital "i" was an "l".

**Comment by agorrod** [comment]
Thanks @keithbostic lgtm

**Pull request merged**